modules/magic_enum/0.9.3/MODULE.bazel

```diff
@@ -0,0 +1,8 @@
+module(
```

Comment:
From looking into the upstream repo, I was glad to see that the `copts` are determined based on the detected compiler, not the target platform. Good job!

I could see https://github.com/bazelbuild/rules_cc/blob/main/cc/compiler/BUILD working a bit better as these conditions apply based on the `compiler` name set on a toolchain and don't require explicit tagging of platforms with references to the constraint values in `@bazel_tools//tools/cpp`.

Reply:
Thanks for the tip! Sounds like I've got a bunch of repos to update now 😅

Reply:
@fmeum I've changed magic_enum to use your recommendation. Would you mind taking a look at it upstream? It works for me, but I want to make sure it will work on the bazel central registry CI https://github.com/Neargye/magic_enum/pull/291